README.md

```diff
@@ -21,6 +21,25 @@ This script will:
 - runs `./scripts/copy-resource.sh` that will copy patched version of code - oss from `./vscode` into `./patched-vscode` folder along with icon(s) and svg(s) from `./resources` folder
 - runs `yarn install` and downloads built in extensions on patched submodule
 
+## Test Execution
+Follow the previous steps to patch and build the project.
```

Comment:
Eventually you should include the patching/building as part of the test script so it's not a bunch of separate steps for the dev to run